docs/devops-guide/docker.md

```diff
@@ -6,15 +6,18 @@ sidebar_label: Docker
 
 ## Quick start
 
-In order to quickly run Jitsi Meet on a machine running Docker and Docker Compose,
-follow these steps:
+In order to quickly run Jitsi Meet on a machine running Docker, make use of Docker Compose and follow these steps:
```

Comment:
I would change it to something like:
`In order to run Jitsi Meet on a machine using Docker and Docker Compose follow these steps:`

Reply:
I was new to Docker, and hadn't heard of Docker Compose. It seemed like an extension to Docker which I didn't want. I appeared to be wrong about the purpose/scope of Compose. So I wanted to specify 'if you want to use Docker, you'll have to use Docker Compose'.

Reply:
Maybe we could link to the docker documentation to explain docker compose: https://docs.docker.com/compose/
It is a seperate tool after all.
Basically it starts several docker containers at once.
Strictly seen it is not necessary, just necessary for this guide.

So maybe we should rewrite this to achieve something like:
`For this guide you need to use Docker and Docker Compose...`